050-Manage/060-migrations-multischema.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Multi Schema Migrations
```

Comment:
```suggestion
title: Multi schema migrations
```

Sentence format to match other docs pages

Reply:
As I read this blog, I'm realizing that this is being productized as multi-schema migrations. We've always used this internally to discuss the feature, but I'm not sure if it'll be confusing externally. What are yours  and @andrew-farries @exekias @tsg on calling this "Reversible schema migrations" or "Simultaneous schema migrations" or "Virtual schema migrations". Just brainstorming here. Part of the problem is we have two was to do migrations at the moment. When there's one, we don't really have to differentiate in the docs. We could also combine into the same migrations page, and differentiate for postgres dbs vs. not.

Reply:
Maybe we can also call it "Online / Live Schema Migrations"? That's the term I am aware of with products that feel the most similar to `pgroll` like https://github.com/github/gh-ost